Source/DataAccessLayer.SqlClient/Views/ViewsXml.cst

```diff
@@ -78,7 +77,7 @@ public string GetSetTransactionIsolationLevelStatement()
 
 public string RemoveCreate(string body)
 {
-	if (string.IsNullOrWhiteSpace(body))
+	if (string.IsNullOrEmpty(body) || body.Trim().Length == 0)
```

Comment:
Are you currently using .NET 3.5?
If so, do you have any plans to upgrade your software?

It would be nice at some point if we can abandon support for some of the super old frameworks.

Reply:
Yes, a project that I have been using it for has the old Codesmith  v5.0.1

Also, NetTeirs.cst itself says `CompilerVersion="v3.5"` so  `string.IsNullOrWhiteSpace` would error out anyway. I tried v4.0, v4.5, v4.7.2 but it errored out saying csc.exe not found. (I have various .net versions installed)

Do we get any benefits, other then using the `string.IsNullOrWhiteSpace`, if we can upgrade to a new version?

Reply:
Why not just use `if (string.IsNullOrEmpty(body.Trim()))`. I'm thinking we may just want to store the trimmed value in a variable that can be used inside of this method.

The CompilerVersion attribute just controls what default namespaces get added to the templates and the compiler version passed to csc (if I remember correctly). This could be easily changed to the latest version and the templates would still work as is and we could use newer features of the runtime (Generator 8.x is compiled against .NET 4.8).

Reply:
`if (string.IsNullOrEmpty(body.Trim()))`

Wouldn't this throw an exception if `body` was `null`?

Reply:
@rollercoaster yes, I was thinking of using body?.Trim() but I guess that's not available here either